app/javascript/controllers/keywords_controller.js

```diff
@@ -23,16 +23,23 @@ export default class extends Controller {
     this.containerTarget.classList.remove('is-invalid')
   }
 
-  checkForEnter(event) {
-    if (event.key === 'Enter') {
-      event.preventDefault() // prevent form submission
-      this.add(event);
+  checkForDuplicates(event) {
+    const input = event.target.closest('input[type="text"]')
```

Comment:
Could you use a stimulus target for this?

Reply:
Yup - done